### PR TITLE
Update desktop-changelog.rst

### PR DESCRIPTION
**Pull request by amyblais**
Adding known issues to the changelog.

### CODE REVIEW

File: source/help/apps/desktop-changelog.rst [hunk 1]
```diff
@@ -60,6 +60,11 @@ Mac
 
 Known Issues
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- When minimizing the app and then maximizing it, it doesn't restore to the right position but "jumps" to a different place in the taskbar. [MM-23195](https://mattermost.atlassian.net/browse/MM-23195)
+- The current window frame and server tabs don't work properly on older Windows OS versions or Linux. [MM-22751](https://mattermost.atlassian.net/browse/MM-22751)
+- Desktop client opens to a blank white Window when using GPO-set teams. [MM-23082](https://mattermost.atlassian.net/browse/MM-23082)
+- Google oAuth no longer works on the Desktop app. [MM-23057](https://mattermost.atlassian.net/browse/MM-23057)
```
Comment (Willyfrog) [inline source/help/apps/desktop-changelog.rst]: this only happening with gmail addresses, gsuite are fine

File: source/help/apps/desktop-changelog.rst [hunk 2]
```diff
@@ -60,6 +60,11 @@ Mac
 
 Known Issues
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- When minimizing the app and then maximizing it, it doesn't restore to the right position but "jumps" to a different place in the taskbar. [MM-23195](https://mattermost.atlassian.net/browse/MM-23195)
+- The current window frame and server tabs don't work properly on older Windows OS versions or Linux. [MM-22751](https://mattermost.atlassian.net/browse/MM-22751)
```
Comment (Willyfrog) [inline source/help/apps/desktop-changelog.rst]: is this an issue or a feature?

Comment (amyblais) [reply source/help/apps/desktop-changelog.rst]: It's a feature ticket but it has been reported by lot of users after v4.4 was released.

Comment (esethna) [reply source/help/apps/desktop-changelog.rst]: @amyblais can we be more specific about "doesn't work properly"

It works fine, it's just not styled consistently with the rest of the OS in Windows 7 or Linux

File: source/help/apps/desktop-changelog.rst [hunk 3]
```diff
@@ -60,6 +60,11 @@ Mac
 
 Known Issues
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- When minimizing the app and then maximizing it, it doesn't restore to the right position but "jumps" to a different place in the taskbar. [MM-23195](https://mattermost.atlassian.net/browse/MM-23195)
```
Comment (esethna) [inline source/help/apps/desktop-changelog.rst]: Sometimes (I don't think we have consistent repro)

**Pull request merged**